test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/HtmlBlockTest/StopsParsingSingleLineBlockAtEOFIfNoEOLReached.stree.txt

```diff
@@ -1,9 +1,9 @@
-Markup block - Gen<None> - 9 - (0:0,0)
-    Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-        SyntaxKind.Transition;[@];
-    MetaCode span - Gen<None> - [:] - SpanEditHandler;Accepts:Any - (1:0,1) - Tokens:1
-        SyntaxKind.Colon;[:];
-    Markup span - Gen<Markup> - [foo bar] - SpanEditHandler;Accepts:Any - (2:0,2) - Tokens:3
-        SyntaxKind.Text;[foo];
-        SyntaxKind.Whitespace;[ ];
-        SyntaxKind.Text;[bar];
+MarkupBlock - [0..9) - FullWidth: 9 - [@:foo bar]
```

Comment:
Eventually we'll want to represent `@:` as its own node type to have a better understanding of the syntax tree. Something like `SingleLineMarkupBlock`